server/src/connectionHandlers/onCompletion.ts

```diff
@@ -454,3 +467,40 @@ function convertExtraSymbolsToCompletionItems (uri: string): CompletionItem[] {
   })
   return completionItems
 }
+
+export const getRangeOfTextToReplace = (document: TextDocument, position: Position): Range => {
```

Comment:
We might want to use this on other places in order to improve the precision of the text being replaced. I'm thinking for example of SRC_URI

Reply:
Agreed